packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx

```diff
@@ -230,7 +234,7 @@ export class TextFieldBase extends BaseComponent<ITextFieldProps, ITextFieldStat
       <div className={this._classNames.root}>
         <div className={this._classNames.wrapper}>
           {onRenderLabel(this.props, this._onRenderLabel)}
-          <div className={this._classNames.fieldGroup}>
+          <div className={this._classNames.fieldGroup} style={{ border: this.state.color }}>
```

Comment:
We should not use the style tag for that when we already have the className attached to this element. Better to have some kind of required boolean flag as a style prop passed to getStyles and do the magic using the appropriate palette or semantic colors.

Reply:
Thanks @Vitalius1 

Shall incorporate the suggestions once we decide on the behavior we'd like to use. Until then, I'll keep the `PR` closed.